src/sentry/api/endpoints/organization_events.py

```diff
@@ -170,6 +171,8 @@ def get(self, request, organization):
 
 
 class OrganizationEventsHeatmapEndpoint(OrganizationEventsEndpointBase):
+    NON_TAG_KEYS = frozenset(['project'])
```

Comment:
Can we support this list of properties in an event? These are the fields we support elsewhere in Sentry. https://github.com/getsentry/sentry/blob/master/src/sentry/utils/snuba.py#L46-L115

Reply:
Gotcha. I'll write up a ticket for this work, and do it in a different pr.